public/app/features/explore/TraceView/TraceViewContainer.tsx

```diff
@@ -50,40 +63,71 @@ export function TraceViewContainer(props: Props) {
     (state: StoreState) => state.explore.panes[props.exploreId]?.datasourceInstance ?? undefined
   );
   const datasourceType = datasource ? datasource?.type : 'unknown';
+  const [ref, { width }] = useMeasure<HTMLDivElement>();
+
+  const links = useMemo(() => {
+    if (!traceProp) {
+      return [];
+    }
+    return getTraceLinks(traceProp);
+  }, [traceProp]);
 
   if (!traceProp) {
     return null;
   }
 
   return (
-    <div className={style.container}>
-      {!config.featureToggles.newTraceViewHeader && (
-        <TracePageSearchBar
-          navigable={true}
-          searchValue={search}
-          setSearch={setSearch}
-          spanFindMatches={spanFindMatches}
-          searchBarSuffix={searchBarSuffix}
-          setSearchBarSuffix={setSearchBarSuffix}
-          focusedSpanIdForSearch={focusedSpanIdForSearch}
-          setFocusedSpanIdForSearch={setFocusedSpanIdForSearch}
-          datasourceType={datasourceType}
-        />
-      )}
-      <TraceView
-        exploreId={exploreId}
-        dataFrames={dataFrames}
-        splitOpenFn={splitOpenFn}
-        scrollElement={scrollElement}
-        traceProp={traceProp}
-        spanFindMatches={spanFindMatches}
-        search={search}
-        focusedSpanIdForSearch={focusedSpanIdForSearch}
-        queryResponse={queryResponse}
-        datasource={datasource}
-        topOfViewRef={topOfViewRef}
-        topOfViewRefType={TopOfViewRefType.Explore}
-      />
+    <div className={style.container} ref={ref}>
+      <PanelChrome
+        padding="none"
+        width={width}
+        title={getTraceName(traceProp.spans)}
+        titleItems={
+          <span className={style.duration}>
```

Comment:
we got pushback in the past about the title "traces" being useless. the alternative here, if we want to text to be bigger, is to put back the "trace" title and add the title inside the panel content as we cannot modify the title size in PanelChrome. opinions?

Reply:
yeah, but we cannot change the title size, so it's either

1. with an "unwanted"  "Trace" title
```
___________________________________________
| Trace                           Buttons
| Trane Title (Big)
```

2. current version:
```
___________________________________________
| Trace Title (small)             Buttons
```
3. or hacky empy string in the title, which would leave a blank space, to verify if possible
```
___________________________________________
|  (empty space)                  Buttons
| Trane Title (Big)
```

Reply:
Sorry maybe I'm not understanding but if we pass in a ReactNode for the trace [title](https://github.com/grafana/grafana/blob/main/packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx#L28) (`lb: HTTP Client`) and style that to be bigger would that work? Not sure why we can't change the title size in PanelChrome? Again not sure if we want to have custom styling for the titles here. 

We can also deemphasize the info around the trace title rather than increasing the title size to emphasize it (if we don't want to add custom title nodes to PanelChrome).

Reply:
@joey-grafana PanelChrome's title can only be a string, not a ReactNode, it's meant to be a string for consistency reasons